model-providers/google/gemini/gemini-common/runtime/pom.xml

```diff
@@ -22,6 +22,11 @@
             <artifactId>quarkus-langchain4j-core</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.auth</groupId>
+            <artifactId>google-auth-library-oauth2-http</artifactId>
+            <version>1.33.1</version>
```

Comment:
I think we already have this version in a property, no?

Reply:
@geoand I did not find it, though this must be also be available the LC4J level and see we import the bom. For now I added the version property to the parent pom